### PR TITLE
Modify init to compare version with github api latest release

### PR DESCRIPTION
**Pull request by RecoX**
When the game is launched it compare the last version with the one in the github's releases, if it is outdated it will launch the ao-autoupdate

It will read the value from the file `INIT/Version.ini`

### CODE REVIEW

File: CODIGO/General.bas [hunk 1]
```diff
@@ -792,24 +792,24 @@ Sub Main()
     MD5HushYo = "0123456789abcdef"  'We aren't using a real MD5
     
     tipf = Config_Inicio.tip
+
```
Comment (RecoX) [inline CODIGO/General.bas]: Cambie esta funcion de lugar

File: CODIGO/General.bas [hunk 2]
```diff
@@ -792,24 +792,24 @@ Sub Main()
...
     'Set resolution BEFORE the loading form is displayed, therefore it will be centered.
     Call Resolution.SetResolution
-    
-    ' Load constants, classes, flags, graphics..
-    LoadInitialConfig
 
-#If Testeo <> 1 Then
-    Dim PresPath As String
-    PresPath = DirGraficos & "Presentacion" & RandomNumber(1, 4) & ".jpg"
+    #If Testeo <> 1 Then
```
Comment (RecoX) [inline CODIGO/General.bas]: Un poco de identacion

File: CODIGO/frmCargando.frm [hunk 3]
```diff
@@ -124,38 +118,30 @@ Private Sub Status_KeyPress(KeyAscii As Integer)
...
-    Close #f
+Private Function CheckIfRunningLastVersion() As Boolean
+    Dim responseGithub As String, versionNumberMaster As String, versionNumberLocal As String
+    Dim JsonObject As Object
+
+    responseGithub = Inet1.OpenURL("https://api.github.com/repos/ao-libre/ao-cliente/releases/latest")
+    Set JsonObject = JSON.parse(responseGithub)
+    
+    versionNumberMaster = JsonObject.Item("tag_name")
+    versionNumberLocal = GetVar(App.path & "\INIT\version.ini", "Cliente", "version")
```
[the remark this answers is not in the document]
Comment (RecoX) [reply CODIGO/frmCargando.frm]: Not really due after do the installation of the update, we open that file and set the version number from github's api.
We just use that file to:
First get the current version we are running it, then we compare it with github's latest release when the update is done after replace all the files we set the value of the last version in it.

So we don't need to do that manually :)

[the remark this answers is not in the document]
Comment (RecoX) [reply CODIGO/frmCargando.frm]: I said nothing, i confuse the file, we need to update that file manually

[the remark this answers is not in the document]
Comment (juandpr) [reply CODIGO/frmCargando.frm]: :+1:

**Pull request merged**